packages/file_selector/file_selector_windows/lib/src/messages.g.dart

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+// Autogenerated from Pigeon (v3.1.0), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name
+// @dart = 2.12
```

Comment:
I realise this was auto-generated, but question for the pigeon generator: is this still necessary now that we're NNBD... BD?

Reply:
Good point, I'll remove that from the generator: https://github.com/flutter/flutter/issues/106526